--- Mothur.source/makefile.orig	2013-09-05 13:56:57.491619749 +1200
+++ Mothur.source/makefile	2013-09-05 13:57:12.190675886 +1200
@@ -28,7 +28,7 @@
 	#CXX = g++44
 	
 	#if you are a mac user use the following line
-	TARGET_ARCH += -arch x86_64
+	# TARGET_ARCH += -arch x86_64
 	
 	#if you using cygwin to build Windows the following line
 	#CXX = x86_64-w64-mingw32-g++
@@ -37,7 +37,7 @@
  	#TARGET_ARCH += -m64 -static
 
 	#if you are a linux user use the following line
-	#CXXFLAGS += -mtune=native -march=native -m64
+	CXXFLAGS += -mtune=native -march=native -m64
 	
 	CXXFLAGS += -DBIT_VERSION 
     FORTRAN_FLAGS = -m64
