--- src/tophat.py.orig	2014-06-11 11:32:33.479650549 +1200
+++ src/tophat.py	2014-06-11 11:35:07.630263760 +1200
@@ -1415,6 +1415,9 @@
         if read_params.color:
             bowtie_header_cmd.append('-C')
 
+        if params.bowtie2:
+            bowtie_header_cmd += ["-x"]
+
         bowtie_header_cmd.extend([idx_prefix, '/dev/null'])
         if noSkip:
            subprocess.call(bowtie_header_cmd,
