--- src/tophat.orig	2013-04-24 10:41:53.312502965 +1200
+++ src/tophat	2013-04-24 10:44:10.410530523 +1200
@@ -1510,7 +1510,8 @@
             die("Error: TopHat requires Bowtie 2.0.5 or later")
     else:
         if bowtie_version[1] < 12 or bowtie_version[2] < 9:
-            die("Error: TopHat requires Bowtie 0.12.9 or later")
+	    if bowtie_version[0] < 1:
+                die("Error: TopHat requires Bowtie 0.12.9 or later")
     th_logp("\t\t  Bowtie version:\t %s" % ".".join([str(x) for x in bowtie_version]))
 
 
