--- fastqc.orig	2015-03-26 09:48:43.159147618 +1300
+++ fastqc	2015-03-26 09:49:39.439504630 +1300
@@ -1,7 +1,7 @@
 #!/usr/bin/perl
 use warnings;
 use strict;
-use FindBin qw($RealBin);
+my $RealBin = '/usr/share/java/fastqc';
 use Getopt::Long;
 
 #####################################################################################
@@ -381,4 +381,4 @@
     Any bugs in fastqc should be reported either to simon.andrews@babraham.ac.uk
     or in www.bioinformatics.babraham.ac.uk/bugzilla/
                    
-    
\ No newline at end of file
+    
