--- seqmonk.orig	2012-05-10 11:03:04.000000000 +0000
+++ seqmonk	2012-10-03 20:18:32.945510742 +0000
@@ -15,10 +15,10 @@
 
 
 if ($ENV{CLASSPATH}) {
-	$ENV{CLASSPATH} .= ":$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
+	$ENV{CLASSPATH} .= ":/usr/share/java/seqmonk:$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
 }
 else {
-	$ENV{CLASSPATH} = "$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
+	$ENV{CLASSPATH} = "/usr/share/java/seqmonk:$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
 }
 
 my @java_args;
@@ -264,4 +264,4 @@
     Any bugs in SeqMonk should be reported either to simon.andrews@babraham.ac.uk
     or in www.bioinformatics.babraham.ac.uk/bugzilla/
                    
-    
\ No newline at end of file
+    
