--- seqmonk.orig	2015-04-24 08:15:02.931051494 +1200
+++ seqmonk	2015-04-24 08:13:44.340624235 +1200
@@ -16,8 +16,8 @@
 
 if ($ENV{CLASSPATH}) {
-	$ENV{CLASSPATH} .= ":$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
+	$ENV{CLASSPATH} .= ":/usr/share/java/seqmonk:$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
 }
 else {
-	$ENV{CLASSPATH} = "$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
+	$ENV{CLASSPATH} = "/usr/share/java/seqmonk:$RealBin:$RealBin/sam-1.32.jar:$RealBin/Jama-1.0.2.jar";
 }
 
