--- tablet.orig	2015-09-02 10:12:01.617007601 +1200
+++ tablet	2015-09-02 10:12:47.939313690 +1200
@@ -5,10 +5,14 @@
 # Uncomment the following line to add additional VM parameters
 # INSTALL4J_ADD_VM_PARAMS=
 
-
 INSTALL4J_JAVA_PREFIX=""
 GREP_OPTIONS=""
 
+JRE_HOME=/usr/lib/jvm/jre-1.8.0-openjdk/jre
+JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk
+JAVA_PATH=$JAVA_HOME
+PATH=$JAVA_PATH/bin:$PATH
+
 read_db_entry() {
   if [ -n "$INSTALL4J_NO_DB" ]; then
     return 1
@@ -357,7 +361,7 @@
 progname=`basename "$prg"`
 cd "$prg_dir"
 prg_dir=`pwd`
-app_home=.
+app_home='/usr/share/java/tablet'
 cd "$app_home"
 app_home=`pwd`
 bundled_jre_home="$app_home/jre"
