--- rsem-plot-transcript-wiggles.orig	2014-03-31 10:33:41.811357933 +1300
+++ rsem-plot-transcript-wiggles	2014-03-31 10:33:58.629512385 +1300
@@ -31,7 +31,7 @@
 my $command = "";
 
 unless (-e "$ARGV[0].transcript.sorted.bam") {
-    $command = $dir."sam/samtools sort $ARGV[0].transcript.bam $ARGV[0].transcript.sorted";
+    $command = $dir."samtools sort $ARGV[0].transcript.bam $ARGV[0].transcript.sorted";
     &runCommand($command);
 }
 unless (-e "$ARGV[0].transcript.readdepth") {
@@ -45,7 +45,7 @@
 	&runCommand($command);
     }
     unless (-e "$ARGV[0].uniq.transcript.sorted.bam") {
-	$command = $dir."sam/samtools sort $ARGV[0].uniq.transcript.bam $ARGV[0].uniq.transcript.sorted";
+	$command = $dir."samtools sort $ARGV[0].uniq.transcript.bam $ARGV[0].uniq.transcript.sorted";
 	&runCommand($command);
     }
     unless (-e "$ARGV[0].uniq.transcript.readdepth") {
