--- rsem-calculate-expression.org	2014-03-31 10:03:00.700098355 +1300
+++ rsem-calculate-expression	2014-03-31 10:21:43.640544941 +1300
@@ -251,7 +251,7 @@
 	}
 
 	# pipe to samtools to generate a BAM file
-	$command .= " | $dir\sam/samtools view -S -b -o $imdName.bam -";
+	$command .= " | $dir\samtools view -S -b -o $imdName.bam -";
     }
     else {
 	$command = $bowtie2_path."bowtie2";
@@ -286,7 +286,7 @@
 	}
 
 	# pipe to samtools to generate a BAM file
-	$command .= " | $dir\sam/samtools view -S -b -o $imdName.bam -";
+	$command .= " | $dir\samtools view -S -b -o $imdName.bam -";
     }
 
     if ($mTime) { $time_start = time(); }
@@ -356,17 +356,17 @@
 }
 
 if ($genBamF) {
-    $command = $dir."sam/samtools sort -@ $nThreads -m $SortMem $sampleName.transcript.bam $sampleName.transcript.sorted";
+    $command = $dir."samtools sort -@ $nThreads -m $SortMem $sampleName.transcript.bam $sampleName.transcript.sorted";
     &runCommand($command);
-    $command = $dir."sam/samtools index $sampleName.transcript.sorted.bam";
+    $command = $dir."samtools index $sampleName.transcript.sorted.bam";
     &runCommand($command);
 
     if ($genGenomeBamF) {
 	$command = $dir."rsem-tbam2gbam $refName $sampleName.transcript.bam $sampleName.genome.bam";
 	&runCommand($command);
-	$command = $dir."sam/samtools sort -@ $nThreads -m $SortMem $sampleName.genome.bam $sampleName.genome.sorted";
+	$command = $dir."samtools sort -@ $nThreads -m $SortMem $sampleName.genome.bam $sampleName.genome.sorted";
 	&runCommand($command);
-	$command = $dir."sam/samtools index $sampleName.genome.sorted.bam";
+	$command = $dir."samtools index $sampleName.genome.sorted.bam";
 	&runCommand($command);
     }
 }
