--- rsem-plot-transcript-wiggles.orig	2013-10-02 12:11:12.260336579 +1300
+++ rsem-plot-transcript-wiggles	2013-10-02 12:11:37.356832816 +1300
@@ -23,7 +23,7 @@
 my $command = "";
 
 unless (-e "$ARGV[0].transcript.sorted.bam") {
-    $command = $dir."sam/samtools sort $ARGV[0].transcript.bam $ARGV[0].transcript.sorted";
+    $command = $dir."samtools sort $ARGV[0].transcript.bam $ARGV[0].transcript.sorted";
     &runCommand($command);
 }
 unless (-e "$ARGV[0].transcript.readdepth") {
@@ -37,7 +37,7 @@
 	&runCommand($command);
     }
     unless (-e "$ARGV[0].uniq.transcript.sorted.bam") {
-	$command = $dir."sam/samtools sort $ARGV[0].uniq.transcript.bam $ARGV[0].uniq.transcript.sorted";
+	$command = $dir."samtools sort $ARGV[0].uniq.transcript.bam $ARGV[0].uniq.transcript.sorted";
 	&runCommand($command);
     }
     unless (-e "$ARGV[0].uniq.transcript.readdepth") {
