--- util/support_scripts/GG_write_trinity_cmds.pl.orig	2014-04-16 09:02:30.303494096 +1200
+++ util/support_scripts/GG_write_trinity_cmds.pl	2014-04-16 09:02:52.992637540 +1200
@@ -71,7 +71,7 @@
     
     my $file = pop @x;
     
-    my $cmd = "$FindBin::Bin/../../Trinity --single \"$file\" --output \"$file.out\" $trin_args ";
+    my $cmd = "/usr/bin/Trinity --single \"$file\" --output \"$file.out\" $trin_args ";
     
     print "$cmd\n";
 }
