--- util/align_and_estimate_abundance.pl.orig	2014-06-11 09:06:14.316752196 +1200
+++ util/align_and_estimate_abundance.pl	2014-06-11 09:06:38.867881826 +1200
@@ -380,7 +380,7 @@
                 
                 &process_cmd("touch $rsem_prefix.rsem.prepped.started");
                 
-                my $cmd = "rsem-prepare-reference --no-polyA --no-bowtie";
+                my $cmd = "rsem-prepare-reference --no-polyA";
                 
                 if ($gene_trans_map_file) {
                     $cmd .= " --transcript-to-gene-map $gene_trans_map_file";
