--- util/run_Trinity_from_samples_file.pl.orig	2016-08-09 08:48:32.375609616 +1200
+++ util/run_Trinity_from_samples_file.pl	2016-08-09 08:51:15.755730430 +1200
@@ -143,12 +143,12 @@
         ## got pairs
     
         ## Run Trinity:
-        $cmd = "$BASEDIR/Trinity --left $reads_ALL_left_fq --right $reads_ALL_right_fq ";
+        $cmd = "/usr/bin/Trinity --left $reads_ALL_left_fq --right $reads_ALL_right_fq ";
         
     }
     else {
         # run left as single
-        $cmd = "$BASEDIR/Trinity --single $reads_ALL_left_fq ";
+        $cmd = "/usr/bin/Trinity --single $reads_ALL_left_fq ";
         
     }
     
