--- util/run_Trinity_edgeR_pipeline.pl.orig	2013-03-04 12:46:02.863258745 +1300
+++ util/run_Trinity_edgeR_pipeline.pl	2013-03-04 12:47:45.723349106 +1300
@@ -8,7 +8,7 @@
 
 ######################################################
 ## Set to base directory of the Trinity installation:
-my $BASEDIR = "$FindBin::Bin/../";
+my $BASEDIR = "/usr/libexec/trinityrnaseq";
 ######################################################
 
 my $usage = <<__EOUSAGE__;
@@ -145,12 +145,12 @@
         ## got pairs
     
         ## Run Trinity:
-        $cmd = "$BASEDIR/Trinity.pl --left $reads_ALL_left_fq --right $reads_ALL_right_fq ";
+        $cmd = "/usr/bin/Trinity.pl --left $reads_ALL_left_fq --right $reads_ALL_right_fq ";
         
     }
     else {
         # run left as single
-        $cmd = "$BASEDIR/Trinity.pl --single $reads_ALL_left_fq ";
+        $cmd = "/usr/bin/Trinity.pl --single $reads_ALL_left_fq ";
         
     }
     
