--- util/support_scripts/write_partitioned_trinity_cmds.pl.orig	2016-08-09 08:44:19.111209899 +1200
+++ util/support_scripts/write_partitioned_trinity_cmds.pl	2016-08-09 08:44:55.571410220 +1200
@@ -71,7 +71,7 @@
     
     my $file = pop @x;
     
-    my $cmd = "$FindBin::RealBin/../../Trinity --single \"$file\" --output \"$file.out\" $trin_args ";
+    my $cmd = "/usr/bin/Trinity --single \"$file\" --output \"$file.out\" $trin_args ";
     
     print "$cmd\n";
 }
